src/plugins/add-cal/add-cal-fr.hbs

```diff
@@ -19,26 +19,31 @@
 
 <span class="wb-prettify all-pre"></span>
 
-<h2>Événement incluant un endroit détaillé</h2>
+<h2>Événement incluant un endroit détaillé avec un conteneur pour le bouton</h2>
 <p>L'exemple suivant présente un événement incluant un emplacement détaillé (nom, adresse, région, province et code postale).</p>
-<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
+<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event" data-addcal-target="addcalTarget">
```

Comment:
- remove class="provisional"
- remove typeof="Event" since its added through the plugin code
```suggestion
<div class="wb-addcal mrgn-bttm-md" data-addcal-target="addcalTarget">
```